templates/ohdsi-webapi/atlas_database.bicep

```diff
@@ -0,0 +1,145 @@
+param location string
+param suffix string
+param odhsiWebApiName string
+param branchName string = 'v2'
+@secure()
+param postgresAdminPassword string
+@secure()
+param postgresWebapiAdminPassword string
+@secure()
+param postgresWebapiAppPassword string
+
+var postgresAdminUsername = 'postgres_admin'
+var postgresWebapiAdminUsername = 'ohdsi_admin_user'
+var postgresWebapiAdminRole = 'ohdsi_admin'
+var postgresWebapiAppUsername = 'ohdsi_app_user'
+var postgresWebapiAppRole = 'ohdsi_app'
+var postgresWebApiDatabaseName = 'atlas_webapi_db'
+var postgresSchemaName = 'webapi'
+var postgresVersion = '14'
+
+
+// Create a PostgreSQL server
+resource postgresServer 'Microsoft.DBforPostgreSQL/flexibleServers@2022-12-01' = {
+  name: 'postgresql-${odhsiWebApiName}-${suffix}'
+  location: location
+  sku: {
+    name: 'Standard_D2s_v3'
+    tier: 'GeneralPurpose'
+  }
+  properties: {
+    version: postgresVersion
+    administratorLogin: postgresAdminUsername
+    administratorLoginPassword: postgresAdminPassword
+    storage: {
+      storageSizeGB: 32
+    }
+    backup: {
+      backupRetentionDays: 7
+      geoRedundantBackup: 'Disabled'
+    }
+    availabilityZone: '3'
+  }
+}
+
+// Allow public access from any Azure service within Azure to this server
+resource allowAccessToAzureServices 'Microsoft.DBforPostgreSQL/flexibleServers/firewallRules@2021-06-01' = {
+  name: 'AllowAllWindowsAzureIps' 
+  parent: postgresServer
+  properties: {
+    startIpAddress: '0.0.0.0'
+    endIpAddress: '0.0.0.0'
+  }
+}
+
+// Create a new PostgreSQL database
+resource postgresDatabase 'Microsoft.DBforPostgreSQL/flexibleServers/databases@2022-12-01' = {
+  name: postgresWebApiDatabaseName
+  parent: postgresServer
+  properties: {
+    charset: 'utf8'
+    collation: 'en_US.utf8'
+  }
+}
+
+// Create a OHDSI users and groupss
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = { 
+    name: 'deploy-script-identity' 
+    location: location 
+} 
+resource runSQLscriptsWithOutputs 'Microsoft.Resources/deploymentScripts@2020-10-01' = { 
+    name: 'runSQLscriptsWithOutputs' 
```

Comment:
`runSQLscriptsWithOutputs` should also be renamed to something more specific to atlas db

Reply:
Done